src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    for (const key in actions[i]) {
```

Comment:
don't use this second cycle. the `action `is an object, so you can just call `action.type` property

Reply:
if yo have questions, ask in the chat